.github/workflows/docker-build.yml

```diff
@@ -1,23 +1,14 @@
 # Candace Savonen Apr 2021
 
-name: Build of Docker
+name: Build Docker
```

Comment:
This on its own is still handy to have to run outside of `pull-request.yml` (I use it often to rebuild images and push them to dockerhub). But it was able to be simplified now that it doesn't need to run on pull request. (the version that runs on pull request is in the pull-request.yml See #448)